content/_index.fr.md

```diff
@@ -14,7 +14,7 @@ menu:
 ## 3 et 4 Septembre 2022
 ### La Dynamo Chambéry
 
-<a class="btn primary btn-lg" style="margin-top: 1em;" href="https://drive.google.com/file/d/1td_9Cr1b2JZvv0bCpOCJNDsEWgVgEp2Y/view?usp=sharing" target="_blank">Devenez sponsor</a>
+<a class="btn primary btn-lg" style="margin-top: 1em;" href="https://github.com/sre-summercamp/sre-summercamp.github.io/raw/main/static/summercamp-2022-deck.pdf" target="_blank">Devenez sponsor</a>
```

Comment:
In this case if the PDF file is under the same repo/site you can use an internal reference like
```html
href="/summercamp-2022-deck.pdf"
```
instead of
```html
href="https://github.com/sre-summercamp/sre-summercamp.github.io/raw/main/static/summercamp-2022-deck.pdf"
```

Reply:
Thanks, done!